client/src/components/fields/RenderTextField.js

```diff
@@ -20,16 +20,23 @@ export const RenderTextField = ({
 
   return (
     <Fragment>
-      {label && <InputFieldLabel label={label} />}
+      <label htmlFor={fieldRest.name}>
```

Comment:
Nitpick: We're using `fieldRest.name` A LOT. We could just pull this out on line 8 via `field: { value, name, ...fieldRest },`.

Reply:
True, I completely missed that